### PR TITLE
arcticdb v1.6.1rc0

### PR DESCRIPTION
**Pull request by mehertz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,14 @@
 {% set name = "arcticdb" %}
-{% set version = "1.5.0" %}
+{% set version = "1.6.1-rc0" %}
+{% set version_suffix = "" %}
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: {{ version }}{{ version_suffix }}
```
Comment (jjerphan) [inline recipe/meta.yaml:7]: Is there a reason for us to keep `version_suffix` or could we remove it?

**Review by jjerphan — APPROVED** [review]
Thank you, @mehertz!

Everything looks in order to me, and I think this PR can be merged once all runs complete!

Ideally, https://github.com/conda-forge/python-xxhash-feedstock/pull/30 should be merged first, but it likely might not so let's keep the pin.

I just have one question.

**Review by jjerphan — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,17 +1,14 @@
 {% set name = "arcticdb" %}
-{% set version = "1.5.0" %}
+{% set version = "1.6.1-rc0" %}
```
[the remark this answers is not in the document]
Comment (mehertz) [reply recipe/meta.yaml:2]: :'( 

Let's just name this manually without the dash for now.

I think it's okay to have pip with dashes and Conda without dashes.

[the remark this answers is not in the document]
Comment (mehertz) [reply recipe/meta.yaml:2]: I've hardcoded the version without the dash.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: 1.6.1rc0
```
Comment (jjerphan) [inline recipe/meta.yaml:6]: This suggestion has to be committed with 2 other ones (1/3)

```suggestion
  # conda-forge does not support `-` as a separator, so we omit it.
  version: {{ version }}{{ version_suffix }}
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,17 +1,13 @@
...
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
```
Comment (jjerphan) [inline recipe/meta.yaml:9]:
```suggestion
  url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}-{{ version_suffix }}.tar.gz
```

**Review by jjerphan — COMMENTED** [review]
This suggestion can be omitted.

**Pull request merged**